app/views/smart_proxies/logs/_modules_card.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="card-pf card-pf-accented card-pf-aggregate-status card-pf-with-action">
+  <h2 class="card-pf-title">
+    <a href="#" data-toggle="tooltip" data-placement="top" title="<%= features.join(', ') %>"><%= n_('%s Features active', '%s Features active', features_started) % features_started %></a>
```

Comment:
No need for this to be an `<a>` as it links nowhere which is confusing. better just a span for the tooltip or add it directly to the `<h2>`.